microros_static_library/library_generation/library_generation.sh

```diff
@@ -26,6 +26,13 @@ pushd firmware/mcu_ws > /dev/null
     # Import user defined packages
     mkdir extra_packages
     pushd extra_packages > /dev/null
+    	USER_CUSTOM_PACKAGES_DIR=/project/microros_component/extra_packages 
+    	if [ -d "$USER_CUSTOM_PACKAGES_DIR" ]; then
+    		cp -R $USER_CUSTOM_PACKAGES_DIR/* .
+		fi
+        if [ -f extra_packages.repos ]; then
```

Comment:
Shouldn't this be in `$USER_CUSTOM_PACKAGES_DIR/extra_packages.repos`?

Reply:
actually the file will be copied to current directory so both will behave same

Reply:
fixed in last commit